proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -332,3 +351,20 @@ message Pool {
     (gogoproto.moretags)   = "yaml:\"bonded_tokens\""
   ];
 }
+
+// InfractionType indicates the infraction type a validator commited.
+enum InfractionType {
```

Comment:
how come this is in staking instead of evidence? I think the evidence module should handle this instead of staking

Reply:
I think it was a decision made by @sainoe after a discussion with @alexanderbez (if I'm not mistaken).

Reply:
@alexanderbez do you remember the outcome? Right now it feels like we are conflating evidence with staking when evidence module exists for this purpose